docs/docs-content/vertex/system-management/enable-non-fips-settings/use-non-fips-addon-packs.md

```diff
@@ -12,79 +12,83 @@ tags: ["vertex", "non-fips"]
 
 
 Palette VerteX provides the following FIPS-compliant infrastructure components in Kubernetes clusters it deploys. Review [FIPS-Compliant Components](../../fips/fips-compliant-components.md) to learn more.
-
-<br />
-
     
 - Operating System (OS)
-
 - Kubernetes
-
 - Container Network Interface (CNI)
-
 - Container Storage Interface (CSI)
 
-As shown in the screenshot below, the FIPS-compliant icon used to indicate full FIPS compliance is displayed next to Palette VerteX infrastructure components in the cluster profile stack. To learn about other icons Palette VerteX applies, refer to [FIPS Status Icons](../../fips/fips-status-icons.md).
+VerteX also provides the flexibility to utilize non-FIPS infrastructure packs. To do this, you must add the **Non-fips pack registry** to the tenant. Our support team will provide you with the registry URL.
```

Comment:
```suggestion
VerteX also provides the flexibility to utilize non-FIPS packs. To do this, you must add the **Non-fips pack registry** to the tenant. Our support team will provide you with the registry URL.
```

Reply:
+1 on this suggestion.

Reply:
I updated the suggestion to remove "also". Now that this page applies to all packs, I think we can state, "VerteX provides the flexibility ....."

Reply:
We can share the URL (it's the same URL),  but they need to talk to our support team to get the credentials. 

- Non-FIPS Packs Registry:  `415789037893.dkr.ecr.us-east-1.amazonaws.com` 
  - Base Path: `production`
  
 - Community Registry: `415789037893.dkr.ecr.us-east-1.amazonaws.com` 
   -  Base Path: `community`

Reply:
Added